data/seed.mssql.js

```diff
@@ -0,0 +1,32 @@
+import mssql from "mssql";
```

Comment:
I'm wondering if adding all of these set up files is necessary for the public use of this or is it really just for our development? Also, should we be checking the test data into our repo, or is it something we can just pull in?